README.md

```diff
@@ -1,16 +1,59 @@
+
+<p align="center">
+<img width="600" alt="Screen Shot 2021-08-05 at 10 33 19" src="https://user-images.githubusercontent.com/63513562/128310730-72ff752e-fe81-4ed5-a32e-d2cbb85311c3.png">
+</p>
+
+When the extension icon is clicked, by default the Willow extension window will be opened in a new dedicated tab. The user may also choose the view Willow on a side panel. This side panel is movable and resizable. 
+
+<p align="center">
+<img width="800" alt="Screen Shot 2021-08-05 at 16 51 49" src="https://user-images.githubusercontent.com/63513562/128361709-ba42f967-e627-447e-be9a-d08d6162f8ce.png">
+</p>
+
+The user can revisit a desired web page by double-clicking on the associated node (or restore any previous session and visit the desired webpage from that session). 
+
+## Discover web site map
+Willow also provides a tool to crawl a particular URL and find all webpages underneath the given page. Willow also displays the number of broken links on a particular page, if any. This feature should be especially useful for web site developers as they can clearly see the navigation patterns from the home page to others as well as any broken links.
+
+<p align="center">
+<img width="600" alt="Screen Shot 2021-08-05 at 11 42 41" src="https://user-images.githubusercontent.com/63513562/128320391-5a54cb2a-426e-4601-a5df-73f8cb329274.png">
+</p>
+
+## Keyword search
+The user may also search a keyword using the Search tool availble in the toolbar. The nodes whose URL or title contain the keyword will be highlighted as a result.
+
+<p align="center">
+<img width="600" alt="Screen Shot 2021-08-06 at 13 12 40" src="https://user-images.githubusercontent.com/63513562/128495405-72105481-7ccf-460f-b1c8-7c0e772a0b0e.png">
+</p>
+
+## Node annotations
+- The *green arrow* on the upper left of a node indicates the current active page in a tab. So, if multiple tabs are opened after the session started, there might be multiple such pages (one per tab).
 
-![image](https://user-images.githubusercontent.com/54668892/114511492-a4b38300-9c40-11eb-8c93-395414d6c30a.png)
+- The *note* icon on the lower right of a node indicates that a note has been added to the web page associated with this node.
 
+<p align="center">
+<img width="200" alt="Screen Shot 2021-08-05 at 13 34 42" src="https://user-images.githubusercontent.com/63513562/128336494-9b0c6e12-91d2-41b4-85fa-c4bc7b8a4185.png">
+</p>
 
+The users are also allowed to change each node's border color, size, etc. for classification purposes.
 
+# Team 
+[Can Cebeci](https://github.com/CanCebeci), [Cem Cebeci](https://github.com/DraculAlucard), [Efe Dagdemir](https://github.com/efedagdemir), [](https://github.com/CanCebeci), [Tuana Turkmen](https://github.com/tuanaturkmen), [Selbi Ereshova](https://github.com/SelbiEreshova), and [Ugur Dogrusoz](https://github.com/ugurdogrusoz) of [Computer Engineering Department](http://www.cs.bilkent.edu.tr) at Bilkent University.
```

Comment:
Can you change CemCebeci's URL to https://github.com/cemcebeci ? The other one is a result of a faulty GitKraken configuration.

Reply:
Also, Sezin Zeydan (https://github.com/sezinzeydan) appears to be missing here.